.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "markdown_metayaml"]
```

Comment:
Yes, it's the current version of the [`mdx_meta_yaml` project that was originally used](https://github.com/joachimneu/pelican-md-yaml/tree/3c46395a237fef433cb85317162b9590864cd81b/md_yaml/mdx_meta_yaml). It's used to do the actual parsing of the markdown file. This plugin basically provides the interop between Pelican and that project.